app/src/main/java/com/example/taskplanner/util/Progress.kt

```diff
@@ -1,5 +1,9 @@
 package com.example.taskplanner.util
 
-enum class Progress(val value: String) {
-    TODO("Todo"), IN_PROGRESS("In Progress"), DONE("Done")
+import com.example.taskplanner.R
+
+enum class Progress(val value: String, val color: Int) {
+    TODO("Todo", R.color.blue_500),
+    IN_PROGRESS("In Progress", R.color.purple),
+    DONE("Done", R.color.green_500)
```

Comment:
rename "progress"  to "status",  because "in progress" is a part of this class as a value.  it will be more understandable for enum to represent "status" of projects

Reply:
please, also move into string resources "todo" "In progress", "done" values

Reply:
done